scripts/makeDeposit.py

```diff
@@ -1,30 +1,49 @@
 # Bailee Segars
 import pandas as pd
+from decimal import *
```

Comment:
```suggestion
from decimal import Decimal
```
import only what we need from module